packages/main/test/specs/ColorPicker.spec.js

```diff
@@ -46,10 +46,10 @@ describe("Color Picker general interaction", () => {
 
 		await alphaInput.click();
 		await browser.keys('Delete');
+		await browser.keys('Backspace');
```

Comment:
do not modify that test

Reply:
This will be modified with color picker changes

Reply:
Why is this test modified?